### ISSUE TITLE
Building SSR - css.map no longer exported?

### ISSUE THREAD

**Issue by joshrayman** [mhkeller/layercake-template#5]
Hello,

Tried building a project with the SSR flag recently and it looks like the `App.render()` is no longer outputting the map, you get this error in pre-render.js

```
TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received null
```

I fixed it for my project by removing the final line of pre-render.js (as I didn't need the styles.css.map file).

```
fs.writeFileSync(path.join(outDir, 'styles.css.map'), css.map, 'utf-8');
```

Happy to open a PR with that change but thought I'd flag it up first as it might be a config/version number issue my end.

npm details:
```
svelte: 3.42.6
layercake: 5.0.0_svelte@3.42.6
```

**Comment by mhkeller** [comment]
I'm transferring this issue over to https://github.com/mhkeller/layercake-template since it has to do with the template

**Comment by joshrayman** [comment]
Doesn't appear to have sourcemap functionality - Firefox figures out linebreaks so the file is readable. I was between using sveltekit and this to do SSR in this project and it required a little more rejigging to convert so I stuck with this for now (I couldn't figure out custom stores with SSR in it).

Dug around the `rollup-plugin-svelte` issues history and it looks like sourcemaps are still being worked out: https://github.com/sveltejs/svelte/pull/6744#issuecomment-925535462

And were removed late last year (I think I noticed this in another project earlier in this year): https://github.com/sveltejs/rollup-plugin-svelte/pull/147/files#

It doesn't appear like it's likely to be returned to the current position (in the svelte.compile output), so I'll create a PR commenting out that line.
